deployment/terraform/create.go

```diff
@@ -317,6 +317,10 @@ func (t *Terraform) updateAppConfig(ip string, sshc *ssh.Client, output *Output)
 		clusterDSN = "postgres://" + t.config.DBUserName + ":" + t.config.DBPassword + "@" + output.DBCluster.ClusterEndpoint + "/" + t.config.ClusterName + "db?sslmode=disable"
 		readerDSN = []string{"postgres://" + t.config.DBUserName + ":" + t.config.DBPassword + "@" + output.DBCluster.ReaderEndpoint + "/" + t.config.ClusterName + "db?sslmode=disable"}
 		driverName = "postgres"
+	case "cockroachdb":
```

Comment:
We don't really need this case. We can just check if the terraform output has DBs, and if not straightaway go for copy-pasting whatever values are given in the datasource, drivername and replicas. This makes things more readable IMO.

You might need to add an extra method - HasDB.